.travis.yml

```diff
@@ -22,6 +22,8 @@ install:
 - npm install
 
 script:
+- jq --version
+- if [[ $(jq -r ".version" package.json) == *"-rc" ]]; then sed -i "s#https://aka.ms/iot-pnp-config-staging-v2.json#https://aka.ms/iot-codegen-cli-for-workbench-staging#g" package.json; else  sed -i "s#https://aka.ms/iot-pnp-config-staging-v2.json#https://aka.ms/iot-codegen-cli-for-workbench#g" package.json; fi

```

Comment:
the codeGenConfigUrl is https://aka.ms/iot-codegen-cli-for-workbench-staging now, only need to replace if it's not rc

Reply:
Done.